controllers/secretagentconfiguration_controller.go

```diff
@@ -97,6 +97,9 @@ func (reconciler *SecretAgentConfigurationReconciler) Reconcile(req ctrl.Request
 	// set the SAC status to inProgress only the first time around.
 	if instance.Status.State == "" {
 		if err := reconciler.updateStatus(ctx, &instance, true, false); err != nil {
+			if k8serror.IsConflict(err) {
+				return ctrl.Result{Requeue: true}, nil
```

Comment:
can you add an info debug print here? something  indicating we couldn't update the status